src/main/java/roomescape/dto/request/MemberCreationRequest.java

```diff
@@ -0,0 +1,22 @@
+package roomescape.dto.request;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import org.hibernate.validator.constraints.Length;
+import roomescape.domain.regex.MemberFormat;
+
+public record MemberCreationRequest(
+        @NotBlank(message = "비어있는 이메일로는 회원가입이 불가능합니다.")
+        @Length(max = 255, message = "[ERROR] 255자를 초과한 값은 허용하지 않습니다.")
+        @Pattern(regexp = MemberFormat.EMAIL, message = "이메일 형식을 제대로 입력해주세요.")
+        String email,
+        @NotBlank(message = "비어있는 비밀번호로는 회원가입이 불가능합니다.")
```

Comment:
빈 줄을 추가해 가독성을 높일 수 있겠네요!

```suggestion
        String email,
        
        @NotBlank(message = "비어있는 비밀번호로는 회원가입이 불가능합니다.")
```

Reply:
확실히 생각하지 못한 측면입니다! 바로 추가해보겠습니다!